python/paddle/fluid/tests/unittests/dygraph_to_static/test_save_inference_model.py

```diff
@@ -44,7 +44,8 @@ def forward(self, x):
 
 
 class TestDyToStaticSaveInferenceModel(unittest.TestCase):
-    def test_save_inference_model(self):
+    # TODO(Aurelius84): disable temporarily, need new save_inference interface
```

Comment:
Why? Is this due to the program is wrapped as dygraph layer so we cannot save inference model directly?

Reply:
If so we should design new save methods. We should investigate whether save_dygraph is enough in our current case.

Reply:
`save_inference_model` requires `executor`, but there is no `executor` in dygraph mode and it is inside `run_program_op`(C++).

The way of saving/ loading parameters of model is consistent with dygraph: `fluid.save_dygraph` and `fluid.load_dygraph`.

Also, we need a better way to implement `save_inference_model`.

Reply:
Sure. I will approve this PR, but we should discuss whether we can delete `save_inference_model` API